### ISSUE TITLE
Gui not being displayed

### ISSUE THREAD

**Issue by NataliaFavila** [Defense-Circuits-Lab/findmycells#42]
Dear team,

I am having issues deploying the gui from findmycells. I am getting the following error message:

<img width="418" alt="Screenshot 2023-12-07 at 12 33 22" src="https://github.com/Defense-Circuits-Lab/findmycells/assets/13204077/c16b5d34-f04a-43f4-8781-a01d6f2fb045">

I am using python version 3.10.12 and ipywidgets version 7.6.5

Any ideas what could be wrong?

Thank you!
Natalia

**Comment by KonKob** [comment]
Dear Natalia, 

Unfortunately I was not able to reproduce the error. Which Jupyter notebook version did you use? 
I would recommend searching stackoverflow, as it seems to be an issue related to the interaction of ipywidgets and Jupyter notebook.
Alternatively you can run findmcells on Google Colab.

Let me know, if you find a solution!

**Comment by NataliaFavila** [comment]
Hello,
thank yo for replying!  I have managed to solve it.
So first I notice that for some reason inside the environment the jupypter widget extension was not enable:

<img width="564" alt="Screenshot 2023-12-08 at 12 04 14" src="https://github.com/Defense-Circuits-Lab/findmycells/assets/13204077/e110880e-2ebc-46b7-9868-8d6e3f2d4626">

To solve it I ran the following two lines of command:

conda install -n base -c conda-forge widgetsnbextension
conda install -n findmycells -c conda-forge ipywidgets

And now the widgets extension is enable and I can launch the gui from a jupyter notebook.
This happened to me in several computers where I tried to install it, and the solution worked both in mac and windows.

<img width="563" alt="Screenshot 2023-12-08 at 12 07 35" src="https://github.com/Defense-Circuits-Lab/findmycells/assets/13204077/5b859543-6162-4439-9e07-1b2d365d5b5f">
